test/run_tests

```diff
@@ -20,4 +34,27 @@ then
   echo "WARNING: '$PYTHON' not found, using 'python' instead."
   PYTHON=python
 fi
+
+if [ -z "$ARGS" ]
+then
+  # No arguments specified: use tox
+  # We use flags to disable tests that use external non tox-installed
+  # software.
+
+  # Check tox
+  tox --version >/dev/null 2>/dev/null
+  if [ ! $? -eq 0 ]
+  then
+    echo "ERROR: tox is not installed."
+    echo "You can still run ./run_tests with arguments: try ./run_tests -h"
+    exit 1
+  fi
+
+  # Run tox
+  export UT_FLAGS="-K tcpdump -K manufdb -K wireshark -K ci_only"
+  export SIMPLE_TESTS="true"
+  PYVER=$(python3 -c "import sys; print('.'.join(sys.version.split('.')[:2]))")
```

Comment:
Why are you enforcing python3 here while the script understand $PYTHON to specify the version to use.

Reply:
Good catch